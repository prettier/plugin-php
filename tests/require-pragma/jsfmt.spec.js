run_spec(__dirname, ["php"], { requirePragma: true, keepEOL: true });
