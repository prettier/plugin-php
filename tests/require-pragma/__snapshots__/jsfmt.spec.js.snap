// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`file-with-alt-pragma-CRLF.php - php-verify: file-with-alt-pragma-CRLF.php 1`] = `
<?php
/**
 * This works too
 *
 * @prettier
 */

echo 'This ' . 'string ' . 'was ' . 'made ' . 'with concatenation.' . 'This ' . 'string ' . 'was ' . 'made ' . 'with concatenation.' . "\\n";
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<?php
/**
 * This works too
 *
 * @prettier
 */

echo 'This ' .
    'string ' .
    'was ' .
    'made ' .
    'with concatenation.' .
    'This ' .
    'string ' .
    'was ' .
    'made ' .
    'with concatenation.' .
    "\\n";

`;

exports[`file-with-alt-pragma-LF.php - php-verify: file-with-alt-pragma-LF.php 1`] = `
<?php
/**
 * This works too
 *
 * @prettier
 */

echo 'This ' . 'string ' . 'was ' . 'made ' . 'with concatenation.' . 'This ' . 'string ' . 'was ' . 'made ' . 'with concatenation.' . "\\n";
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<?php
/**
 * This works too
 *
 * @prettier
 */

echo 'This ' .
    'string ' .
    'was ' .
    'made ' .
    'with concatenation.' .
    'This ' .
    'string ' .
    'was ' .
    'made ' .
    'with concatenation.' .
    "\\n";

`;

exports[`file-with-pragma-CRLF.php - php-verify: file-with-pragma-CRLF.php 1`] = `
<?php
/**
 * This turns on prettier formatting if --require-pragma is passed
 *
 * @format
 */

echo 'This ' . 'string ' . 'was ' . 'made ' . 'with concatenation.' . 'This ' . 'string ' . 'was ' . 'made ' . 'with concatenation.' . "\\n";
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<?php
/**
 * This turns on prettier formatting if --require-pragma is passed
 *
 * @format
 */

echo 'This ' .
    'string ' .
    'was ' .
    'made ' .
    'with concatenation.' .
    'This ' .
    'string ' .
    'was ' .
    'made ' .
    'with concatenation.' .
    "\\n";

`;

exports[`file-with-pragma-LF.php - php-verify: file-with-pragma-LF.php 1`] = `
<?php
/**
 * This turns on prettier formatting if --require-pragma is passed
 *
 * @format
 */

echo 'This ' . 'string ' . 'was ' . 'made ' . 'with concatenation.' . 'This ' . 'string ' . 'was ' . 'made ' . 'with concatenation.' . "\\n";
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<?php
/**
 * This turns on prettier formatting if --require-pragma is passed
 *
 * @format
 */

echo 'This ' .
    'string ' .
    'was ' .
    'made ' .
    'with concatenation.' .
    'This ' .
    'string ' .
    'was ' .
    'made ' .
    'with concatenation.' .
    "\\n";

`;

exports[`file-without-pragma-CRLF.php - php-verify: file-without-pragma-CRLF.php 1`] = `
<?php
/**
 * No pragma, no prettier (assuming --require-pragma)
 */

echo 'This ' . 'string ' . 'was ' . 'made ' . 'with concatenation.' . 'This ' . 'string ' . 'was ' . 'made ' . 'with concatenation.' . "\\n";
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<?php
/**
 * No pragma, no prettier (assuming --require-pragma)
 */

echo 'This ' . 'string ' . 'was ' . 'made ' . 'with concatenation.' . 'This ' . 'string ' . 'was ' . 'made ' . 'with concatenation.' . "\\n";

`;

exports[`file-without-pragma-LF.php - php-verify: file-without-pragma-LF.php 1`] = `
<?php
/**
 * No pragma, no prettier (assuming --require-pragma)
 */

echo 'This ' . 'string ' . 'was ' . 'made ' . 'with concatenation.' . 'This ' . 'string ' . 'was ' . 'made ' . 'with concatenation.' . "\\n";
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<?php
/**
 * No pragma, no prettier (assuming --require-pragma)
 */

echo 'This ' . 'string ' . 'was ' . 'made ' . 'with concatenation.' . 'This ' . 'string ' . 'was ' . 'made ' . 'with concatenation.' . "\\n";

`;
